src/pages/about/_our-history.js

```diff
@@ -0,0 +1,385 @@
+import React from 'react'
+import styled from 'styled-components'
+import { CarouselProvider, Slider, Slide, ButtonBack, ButtonNext, Dot } from 'pure-react-carousel'
+import 'pure-react-carousel/dist/react-carousel.es.css'
+import { Wrapper, Container } from 'components/containers'
+import { Header, Image, Text } from 'components/elements'
+import { localize } from 'components/localization'
+import ChevronRight from 'images/svg/fill-chevron.svg'
+import Binary from 'images/svg/binary-blur.svg'
+import Deriv from 'images/svg/deriv-blur.svg'
+import device from 'themes/device'
+
+const SliderWrapper = styled.div`
+    width: 100%;
+    position: relative;
+    padding-bottom: 6rem;
+
+    @media ${device.laptopLC} {
+        padding-bottom: 0;
+    }
+`
+
+const SubHeader = styled(Header)`
+    margin-top: 1.6rem;
+    margin-bottom: 8rem;
+`
+
+const StyledContainer = styled(Container)`
+    position: relative;
+`
+
+const ChevronLeft = styled(ChevronRight)`
+    transform: rotate(180deg);
+`
+
+const StyledWrapper = styled(Wrapper)`
+    background: var(--color-grey-8);
+    padding: 8rem 0;
+    position: relative;
+
+    @media ${device.tabletS} {
+        display: none;
+    }
+`
+
+const StyledSlide = styled.div`
+    display: flex;
+    flex-direction: row;
+    max-width: 79.2rem;
+    align-items: center;
+    margin: 4rem auto;
+    width: 100%;
+`
+
+const StyledSlideText = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`
+
+const StyledDotsWrapper = styled.ul`
+    width: 100%;
+    border-top: 4px solid var(--color-green);
+    display: flex;
+    justify-content: center;
+    position: relative;
+
+    &::after {
+        content: '';
+        width: 1.2rem;
+        height: 1.2rem;
+        background: var(--color-black);
+        border-radius: 50%;
+        position: absolute;
+        left: 0;
+        bottom: 0.8rem;
+    }
+    &::before {
+        content: '';
+        width: 1.2rem;
+        height: 1.2rem;
+        background: var(--color-black);
+        border-radius: 50%;
+        position: absolute;
+        right: 0;
+        bottom: 0.8rem;
+    }
+
+    @media ${device.laptopLC} {
+        display: none;
+    }
+`
+
+const StyledDot = styled.li`
+    margin: 0 3rem;
+    position: relative;
+
+    & .carousel__dot {
```

Comment:
maybe we can add a comment here and mention that we are overriding the library styling